lib/ansible/modules/cloud/azure/azure_rm_webapp.py

```diff
@@ -253,6 +253,16 @@
             version: 6.6
           - name: "php"
             version: "7.0"
+    
```

Comment:
Checks failed due to this blank line. It follows other examples and I don't think it is an error.

Reply:
I think here it compains not because of the blank line, but because there are a few spaces in that blank line. so if you remove the spaces it should be fine.